Mathlib/MeasureTheory/Group/Measure.lean

```diff
@@ -761,6 +784,32 @@ theorem isHaarMeasure_map [BorelSpace G] [TopologicalGroup G] {H : Type _} [Grou
 #align measure_theory.measure.is_haar_measure_map MeasureTheory.Measure.isHaarMeasure_map
 #align measure_theory.measure.is_add_haar_measure_map MeasureTheory.Measure.isAddHaarMeasure_map
 
+/-- The image of a Haar measure under map of scalar multiplication is again a Haar measure. -/
+@[to_additive isHaarMeasure_map_radd
+"The image of a Haar measure under map of scalar addition is again a Haar measure"]
+theorem isHaarMeasure_map_smul {α} [BorelSpace G] [TopologicalGroup G] [T2Space G] [SMul α G]
+    [SMulCommClass α G G] [MeasurableSpace α] [MeasurableSMul α G] [ContinuousConstSMul α G] (a : α)
+    (ha_surj : Surjective (a • · : G → G)) (ha_prop : Tendsto (a • ·) (cocompact G) (cocompact G)):
```

Comment:
I'd be tempted to make this slightly less general but easier to use:
```suggestion
theorem isHaarMeasure_map_smul {α} [Group G] [BorelSpace G] [TopologicalGroup G] [T2Space G] [MulAction α G]
    [SMulCommClass α G G] [MeasurableSpace α] [MeasurableSMul α G] [ContinuousConstSMul α G] (a : α) :
```
I believe you can then use `Homeomorph.smul` instead of `Homeomorph.mulRight`.

Reply:
How about asking (a • ·) to be a homeomorphism?

Reply:
I suspect that's not a useful generalization; if that's a homomorphism then you can pretty much always interpret `a` as an element of `Units α` and then my `Group` version applies.

Reply:
Maybe @urkud has an opinion here?

Reply:
I added it, could you check the PR again? @eric-wieser

Reply:
I think having that for a group action is perfectly fine.